### DISCUSSION TITLE
Supabase-Dart subscription only gets CLOSED event.

### DISCUSSION THREAD

**Discussion by happy-san** [supabase/supabase#777] [Questions]
Following the [example](https://github.com/supabase/supabase-dart/blob/4215aca96798490890602742468e7f3a3eb14f25/example/main.dart) given at pub.dev of supabase and doing this addition to it,

```dart
 Future.delayed(Duration(seconds: 10)).then((_) {
  // remember to remove subscription
  client.removeSubscription(subscription1);
  client.removeSubscription(subscription2);
 });
```
I did receive the events of insertion and deletion during those 10 seconds. Now I'm trying to make a TODO list app using supabase and following the warning of `removing the subscription` this is what I came up with:

```dart
  @override
  void initState() {
    WidgetsBinding.instance.addObserver(LifecycleEventHandler(
        detachedCallBack: () => _removeSubs(),
        resumeCallBack: () => _addSubs()));

    _addSubs();
    super.initState();
  }

```
[full source](https://github.com/happy-san/todo_supabase/blob/dev/lib/main.dart)

The issue I'm facing is that both `_subOnCountriesDelete` and `_subOnCountriesInsert` only return the following:

```yaml
I/flutter ( 5976): event: CLOSED error: null
I/flutter ( 5976): event: CLOSED error: null
```

Please let me know if there's a mistake from my side.

**Comment by kiwicopple** [comment]
This one may be a bug. Could you please open an issue on the `supabase-dart` repo? Thanks @happy-san !
